metainfo2.txt

```diff
@@ -48,13 +48,13 @@ skipFileCopyCrc = "true"
 skipCheckVariant = "true"
 skipCheckRegion = "true"
 FinalScript = "./Toolbox/final/finalScript.sh"
-FinalScriptChecksum = "e26f12b8d3319fcfeb6417f47f2aadd70914fd1e"
+FinalScriptChecksum = "5478e2177180c1654b2fbe2ea307078eb1ea0a26"
 FinalScriptMaxTime = "30"
 FinalScriptName = "Final Script"
 
 [Toolbox\final\Dir]
-FileSize = "1935"
-CheckSum = "a3f6022d4e419b59fb65f1c9ba8b202f8e2ea5c0"
+FileSize = "2613"
+CheckSum = "ABF72681C36872252474CF7FF9A231FC25FCBE5B"
```

Comment:
Also this checksum is in upper cases. Will work but could also relate to some script error 😆

Reply:
I had to make a choice on upper vs lower in my tool - compared to the original in the firmware packs I looked at upper seemed more common.

Either way, both upper and lower work fine as long as all the checksums match - which my tool guarantees

Fyi current tool is here: https://github.com/andrewleech/mib2-toolbox/tree/exe_tools/Tools